README.md

```diff
@@ -1,176 +1,61 @@
-* **gov2** for the latest version of the AWS SDK for Go (version 2)
-* **javav2** for the latest version of the AWS SDK for Java (version 2)
-* **javascriptv3** for the latest version of the AWS SDK for JavaScript (version 3)
-* **php** for the latest version of the AWS SDK for PHP (version 3)
-* **python** for the latest version of the AWS SDK for Python (Boto3)
-* **ruby** for the latest version of the AWS SDK for Ruby (version 3)
-
-
-## Examples for SDKs currently in preview 
-* **.kotlin_alpha** for the alpha version of the AWS SDK for Kotlin
-* **swift** for the preview release of the AWS SDK for Swift
-* **rust_dev_preview** for the developer preview version of the AWS SDK for Rust
-
-## Examples for SDKs that have been deprecated
-
-Code examples for previous AWS SDK versions are archived in this repository but are no longer maintained. These include the following:
-
-* **dotnet** for versions of the AWS SDK for .NET prior to version 3.5
-* **go** for AWS SDK for Go version 1
-* **java** for AWS SDK for Java version 1
-* **javascript** for AWS SDK for JavaScript version 2
-
-As AWS SDK major version numbers increment, this repository will more consistently reflect the version numbers in these folders.
-For example, imagine that the AWS SDK for Ruby moves to a version 4.
-A new rubyv4 folder is added. In this scenario, if AWS officially announces that version 3 of the AWS SDK for Ruby is no longer supported,
-then the previous ruby folder is deleted.
+This repository contains code examples that demonstrate how to use the [AWSK SDK's](https://aws.amazon.com/developer/tools/) to interact with [AWS services](https://aws.amazon.com/products).
+
+Many examples are injected into the [AWS Documentation](https://docs.aws.amazon.com).
+
+## How this repository is organized
+Code examples for each language's SDK can be found within the following sub-directories:
+
+| SDK        | folder                                | SDK version | SDK status  |
+|------------|---------------------------------------|-------------|-------------|
+| .NET       | [dotnetv3/](dotnetv3)                 | 3.5+        | ![[]](https://img.shields.io/badge/-GA-blue)           |
+| .NET       | [dotnet/](dotnet)                     | <3.5        | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| C++        | [cpp/](cpp)                           | 1           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [gov2/](gov2)                         | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [go/](go)                             | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| Java       | [javav2/]()                           | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Java       | [java/](java)                         | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
```

Comment:
This is pencilled in for deprecation, AFIK, but not actually deprecated. Where are you sourcing this info.?

Reply:
Also, are the semiotics of Blue vs. Red enough to indicate 'Active' vs 'Deprecated'?